demos/browser/server.js

```diff
@@ -192,9 +192,43 @@ function serve(host = '127.0.0.1:8080') {
         }).promise();
         respond(response, 200, 'application/json', JSON.stringify({}));
       } else if (request.method === 'POST' && requestUrl.pathname === '/startCapture') {
+
+        const body = await handleStartCaptureRequest(request);
+        const requestBody = JSON.parse(body);
+
+        let attendeeIds = [];
+        if (requestBody.attendeeIds != undefined) {
+          attendeeIds = requestBody.attendeeIds
+        }
+
+        let chimeSDKConfig = {
+          ArtifactsConfiguration: {
+            Audio: {
+              MuxType: "AudioWithActiveSpeakerVideo"
+            },
+            Video: {
+              State: "Enabled",
+              MuxType: "VideoOnly"
+            },
+            Content: {
+              State: "Enabled",
+              MuxType: "ContentOnly"
+            }
+          }
+        }
+      
+        if (attendeeIds.length > 0) {
+          chimeSDKConfig["SourceConfiguration"] = {
+            SelectedVideoStreams : {
```

Comment:
Is it a new feature recently released?

Reply:
This feature was launched with Media Capture pipelines, but the support for selection of attendees was not added to the meeting demo. This code review intends to add that support to the Meeting Demo.

Reference to the CreateMediaCapturePipeline: https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMediaCapturePipeline.html